website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -129,14 +129,6 @@ spec:
         kubernetes.io/arch: amd64
 ```
 
-### Operating System

```

Comment:
we still need this

Reply:
What value does it add? We don't really support OS, why advertise it?

Reply:
for docs, it is better to answer in the negative.

for pr, it is better to be silent.

Reply:
Sorry, I don't think I'm used to the communication style of this group XD Can you translate this for me, Geoff?

Reply:
If a customer comes looking at the FAQs to see if we support Windows, they should find this section that says we only support Linux currently.

Reply:
Fair enough. Will revert.